resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/DriverCommandFeatureStep.scala

```diff
@@ -67,7 +67,7 @@ private[spark] class DriverCommandFeatureStep(conf: KubernetesDriverConf)
     // re-write primary resource, app jar is also added to spark.jars by default in SparkSubmit
     // no uploading takes place here
     val newResName = KubernetesUtils
-      .renameMainAppResource(resource = res, shouldUploadLocal = false)
+      .renameMainAppResource(res, Option(conf.sparkConf), false)
```

Comment:
It is a common practice in Scala to use the name for boolean arguments, i.e. `shouldUploadLocal = false`

Reply:
+1 for @martin-g 's comment. It's also recommended in Spark community, too.

Reply:
BTW, I'm not sure why you need to add config here because we don't use it when `shouldUploadLocal` is `false`, @dcoliversun .

https://github.com/apache/spark/blob/a139ede8fc31c8098b153b6dad31943272d24522/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala#L299-L303

Reply:
Thanks for your comments.
To @martin-g , I add name for boolean on https://github.com/apache/spark/pull/36044/commits/7dd1d0517b21ab277fcd2db785b3f2ce397c03c3. Please review~
To @dongjoon-hyun , it's not necessary to add Spark Config. My original intention is to keep consistent with the deleted code signature. Now I choose to delete it to reduce unnecessary misunderstandings on https://github.com/apache/spark/pull/36044/commits/7dd1d0517b21ab277fcd2db785b3f2ce397c03c3 😁